static/app/components/events/interfaces/spans/header.tsx

```diff
@@ -413,101 +417,124 @@ class TraceViewHeader extends Component<PropType, State> {
-            const {dividerPosition} = dividerHandlerChildrenProps;
-            return (
-              <Fragment>
-                <OperationsBreakdown
-                  style={{
-                    width: `calc(${toPercent(dividerPosition)} - 0.5px)`,
-                  }}
-                >
-                  {this.props.event && (
-                    <OpsBreakdown
-                      operationNameFilters={this.props.operationNameFilters}
-                      event={this.props.event}
-                      topN={3}
-                      hideHeader
-                    />
-                  )}
-                </OperationsBreakdown>
-                <DividerSpacer
-                  style={{
-                    position: 'absolute',
-                    top: 0,
-                    left: `calc(${toPercent(dividerPosition)} - 0.5px)`,
-                    height: `${MINIMAP_HEIGHT + TIME_AXIS_HEIGHT}px`,
-                  }}
-                />
-                <ActualMinimap
-                  spans={this.props.spans}
-                  generateBounds={this.props.generateBounds}
-                  dividerPosition={dividerPosition}
-                  rootSpan={this.props.rootSpan}
-                />
-                <CursorGuideHandler.Consumer>
-                  {({
-                    displayCursorGuide,
-                    hideCursorGuide,
-                    mouseLeft,
-                    showCursorGuide,
-                  }) => (
-                    <RightSidePane
-                      ref={this.props.minimapInteractiveRef}
-                      style={{
-                        width: `calc(${toPercent(1 - dividerPosition)} - 0.5px)`,
-                        left: `calc(${toPercent(dividerPosition)} + 0.5px)`,
-                      }}
-                      onMouseEnter={event => {
-                        displayCursorGuide(event.pageX);
-                      }}
-                      onMouseLeave={() => {
-                        hideCursorGuide();
-                      }}
-                      onMouseMove={event => {
-                        displayCursorGuide(event.pageX);
-                      }}
-                      onMouseDown={event => {
-                        const target = event.target;
-
-                        if (
-                          target instanceof Element &&
-                          target.getAttribute &&
-                          target.getAttribute('data-ignore')
-                        ) {
-                          // ignore this event if we need to
-                          return;
-                        }
-
-                        this.props.dragProps.onWindowSelectionDragStart(event);
-                      }}
-                    >
-                      <MinimapContainer>
-                        {this.renderFog(this.props.dragProps)}
-                        {this.renderCursorGuide({
-                          showCursorGuide,
+      <ProfileContext.Consumer>
```

Comment:
A lot of this diff is because I wrapped the existing components in this `ProfileContext.Consumer`